templates/repo/issue/list.tmpl

```diff
@@ -2,6 +2,88 @@
 <div role="main" aria-label="{{.Title}}" class="page-content repository issue-list">
 	{{template "repo/header" .}}
 	<div class="ui container">
+
+	{{if .PinnedIssues}}
+		<div id="issue-pin-div" {{if .IsRepoAdmin}}data-is-repo-admin{{end}} style="display: flex; flex-grow: 1; flex-wrap: wrap;">
+			{{range .PinnedIssues}}
+				<div class="card pinned-issue-card" data-move-url="{{$.Link}}/pin_move" data-issue-id="{{.ID}}">
+					{{if eq $.Project.CardType 1}}
+						<div class="card-attachment-images">
+							{{range (index $.issuesAttachmentMap .ID)}}
+								<img src="{{.DownloadURL}}" alt="{{.Name}}">
+							{{end}}
+						</div>
+					{{end}}
+					<div class="content gt-p-0">
+						<div class="header">
+							<span class="gt-dif gt-ac gt-vm {{if .IsClosed}}red{{else}}green{{end}}">
+								{{if .IsPull}}
+									{{if .PullRequest.HasMerged}}
+										{{svg "octicon-git-merge" 16 "text purple"}}
+									{{else}}
+										{{if .IsClosed}}
+											{{svg "octicon-git-pull-request" 16 "text red"}}
+										{{else}}
+											{{svg "octicon-git-pull-request" 16 "text green"}}
+										{{end}}
+									{{end}}
+								{{else}}
+									{{if .IsClosed}}
+										{{svg "octicon-issue-closed" 16 "text red"}}
+									{{else}}
+										{{svg "octicon-issue-opened" 16 "text green"}}
+									{{end}}
+								{{end}}
+							</span>
+							<a class="project-board-title gt-vm" href="{{.Link}}">
```

Comment:
`project-board-title`?

Reply:
Yes, this a a copy/paste error. I will copy it under a new name.